benchmarks/benchmark_throughput.py

```diff
@@ -540,6 +557,23 @@ def main(args: argparse.Namespace):
                         action='store_true',
                         default=False,
                         help="Disable decoupled async engine frontend.")
+    parser.add_argument("--weights-load-device",
+                        type=str,
+                        default=None,
+                        choices=["cuda", "neuron", "hpu", "cpu"],
```

Comment:
Why the need to add other than cpu and hpu?

Reply:
I didn't want to block other paths that are theoretically possible, based on the [EngineArgs defaults](https://github.com/HabanaAI/vllm-fork/blob/habana_main/vllm/engine/arg_utils.py#L277). I see that currently this value is based on the `DEVICE_OPTIONS`, so I can change to this.